schedule/qam/common/15-common_base_installation_ay.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: 12-common_base_installation_autoyast
```

Comment:
shouldn't this be called `15-...` ? @rfan1

Reply:
YES, YES

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/16996